src/sql/workbench/contrib/notebook/browser/notebook.component.ts

```diff
@@ -572,6 +577,8 @@ export class NotebookComponent extends AngularDisposable implements OnInit, OnDe
 			if (!isUndefinedOrNull(endCell)) {
 				endIndex = firstIndex(codeCells, c => c.id === endCell.id);
 			}
+
+			this.model.trustedMode = true;
```

Comment:
You're already doing this in runCell so aren't the changes to the trustedMode here unnecessary? Calling runCell should automatically update the trustedMode right?